index.html

```diff
@@ -3,28 +3,34 @@
   <head>
     <meta charset="UTF-8" />
     <title>Linkbrary</title>
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og-title" content="Linkbrary" />
+    <meta property="og-description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요" />
+    <meta property="og:image" content="/images/preview.jpg" />
+    <meta property="op:url" content="941227.netlify.app" />
```

Comment:
```suggestion
    <meta property="op:url" content="https://941227.netlify.app" />
```
url은 http프로토콜까지 명시해주는것이 권장됩니다. 그래서 위처럼 url 경로를 모두 적어주시면 좋을것 같습니다. http프로토콜은 https://developer.mozilla.org/ko/docs/Web/HTTP/Overview 이런 문서 가볍게 훑어보셔도 좋을 것 같습니다!